from app.exceptions.repositories.base import BaseRepoError


class AttributeRepoError(BaseRepoError):
    pass
